main.js

```diff
@@ -382,70 +390,108 @@ function createObjects(onConnect) {
             disconnectTimeout = setTimeout(() => {
                 connected = false;
                 disconnectTimeout = null;
-                logger.warn('host.' + hostname + ' Slave controller detected disconnection. Stop all instances.');
+                logger.warn(hostLogPrefix + ' Slave controller detected disconnection. Stop all instances.');
                 stopInstances(true, () => {
                     // if during stopping the DB has connection again
                     if (connected && !isStopping) {
+                        logger.warn(hostLogPrefix + ' Slave controller has connection again ... restarting');
                         getInstances();
                         startAliveInterval();
                         initMessageQueue();
                     }
                 });
-            }, config.objects.connectTimeout || 2000);
+            }, (config.objects.connectTimeout || 2000) + (!compactGroupController ? 10 : 0));
```

Comment:
Hä?

Reply:
There was a timing thinigy where the main host controller should behave slighly different. I schanged to 500ms ... will check while final testing - and ad a comment